analyzer/src/funTest/assets/projects/synthetic/python-inspector-expected-output.yml

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
Commit message: Please use a proper title like "PipFunTest: Add a test for python-inspector".